docs/application/web/get-started/wearable/first-app.md

```diff
@@ -246,51 +246,49 @@ To run the application on the emulator:
+            ![Enable Wi-Fi](media/testing_your_app_on_gear3.png) ![Select Wi-Fi AP](media/testing_your_app_on_gear4.png)
 
-            Note the IP address the device is using.
+        -   Go to **Wi-Fi networks**, and select a Wi-Fi AP.
+
+            ![Select Wi-Fi AP](media/testing_your_app_on_gear6.png) ![IP Address](media/testing_your_app_on_gear7.png)
 
-            ![Switch on Wi-Fi](media/emulator_target_wifi.png)
+            Note the IP address the device is using.
 
-        -   Go to **Settings \> Gear info**, and switch on the debugging mode.
+        -   Go to **About watch > Debugging**, and switch on the debugging mode.
 
-            ![Switch on debugging](media/emulator_target_debug.png)
+            ![Switch on debugging](media/testing_your_app_on_gear1.png) ![Debugging menu](media/testing_your_app_on_gear2.png) 
 
     <a name="remote_device"></a>
 
     2.  Use the Remote Device Manager to connect the wearable device:
-        1.  In the **Device Manager**, click the **Remote Device Manager** button.
 
-            ![Launch the Remote Device Manager](media/remote_conn_mgr.png)
-
-        2.  In the **Remote Device Manager** window, click **+**.
+        1.  In the **Device Manager**, launch the Remote Device Manager by clicking the related icon.
 
-            ![Add new device](media/remote_conn_new_ww.png)
+            ![Launch the Remote Device Manager](media/remote_conn_mgr.png)
 
-        3.  In the **Add Device** window, enter the device and network details (use the IP address you noted before), and click **Add**.
+        2.  In the **Remote Device Manager** window, click the **Scan** icon(![Scan icon](media/remote_conn_scan_icon.png)).
 
-            ![Define device details](media/remote_conn_add_ww.png)
+            ![Scan devices](media/remote_conn_scan.png)
 
-        4.  In the **Remote Device Manager** window, switch the new device on by clicking the switch under **Connect**.
+        3.  In the **Remote Device Manager** window, switch the new device on by clicking the switch under **Connection**.
 
-            ![Connect the device](media/remote_connect_ww.png)
+            ![Connect the device](media/remote_conn_device.png)
 
-    3.  The device asks for user confirmation. To allow Gear to read log data, copy files to and from your computer, and install the application manually, click the accept mark.
+    3.  The device asks for user confirmation. To allow your device to read log data, copy files to and from your computer, and install the application manually, click the accept mark.
 
-        ![Allow Gear to access data](media/remote_allow_gear_ww.png)
+        ![Allow Gear to access data](media/remote_allow_gear.png)
 
     4.  In the **Device Manager**, confirm that the device is connected (shown in the device list).
 
-        ![Device is connected](media/remote_connected_ww.png)
+        ![Device is connected](media/remote_connected_w.png)
```

Comment:
I'm confused..
In the original image remote_connected_ww.png, the platform version is wearable-5.0.
In this updated new image, wearable-4.0.0.0 is used.

However in the app_run_multiple_emulators.png which appears later,
version 5.0 is used again.

So in the images, 4.0 and 5.0 are mixed.